release_notes.md

```diff
@@ -9,3 +9,7 @@
 - Update Node.js Worker Version to [3.8.1](https://github.com/Azure/azure-functions-nodejs-worker/releases/tag/v3.8.1)
 - Update PowerShell 7.4 Worker Version to [4.0.2930](https://github.com/Azure/azure-functions-powershell-worker/releases/tag/v4.0.2930)
 - Update WebJobsScriptHostService to remove hardcoded sleep during application shut down (#9520)
+- Implement host configuration property for handling pre-cancelled invocation requests (#9523)
+  - If a worker supports CancellationTokens, cancelled invocations will now be sent to the worker by default
+    - Customers can opt-out of this behavior by setting `SendCanceledInvocationsToTheWorker` to `false` in host.json
```

Comment:
Can you share sample scenarios where customers would want cancelled invocations to be sent to the worker versus when they would like to opt-out? Feel free to point me to any docs link if this is already documented.